screens/Schedule.js

```diff
@@ -6,15 +6,14 @@ import { SearchBar } from 'react-native-elements';
 import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
 import { faPlusCircle, faStar, faSearch } from '@fortawesome/free-solid-svg-icons';
 
-const makeEvent = (title, description) => {
-    return { title: title, desc: description };
+const makeEvent = (title, description, tags) => {
+    return { title: title, desc: description, tags: tags };
 }
 
 const eventProps = [
-    makeEvent("Event 1", "Really cool event."),
-    makeEvent("Event 2", "Even cooler event."),
-    makeEvent("Event 3", "This event will blow your socks off.")]
-
+    makeEvent("Event 1", "Really cool event.", [{key: "Tag1"}, {key: "Tag2"}, {key: "Tag3"}]),
```

Comment:
why did you decide to use an array of dictionaries with `key` fields over an array of tags?

Reply:
I kinda just looked up how to display a list of items and found that using the component `FlatList` was the way to go, and `FlatList` accepts an array of items labeled as keys.

Reply:
what I went with: https://facebook.github.io/react-native/docs/using-a-listview

Reply:
gotcha